### PR TITLE
[SPRINT 255][MOB-1533][Android SDK] Kotlin Serialization Module - User

### PR DESCRIPTION
**Pull request by cjae**
## What did I do?
* Create a User model using Kotlin serialization
* Create Unit test 
* Clean test folder structure
---

## Testing
* UserTest.kt
---

## Deployment/Dependency Checklist:

* [ ] [List_out_any_new_ENV_vars]
* [ ] [List_out_any_back_end_dependencies]
---
## PR notes contain:
* [ ] Checklist for any deployment dependencies (such as new env vars, back-end dependencies, etc)
## PR Checklist:
* [ ] **I tested this**
* [x] I updated from the base branch (usually `latest`)
* [ ] I attached screenshot(s) (if there were UI changes)
* [x] I used the following PR title format: "[SPRINT-XX][fat-xx] BLAH"
  * Add "[EFS]" to the PR title if it was in the ticket title, e.g. "[SPRINT-XX][fat-xx][EFS]"
* [x] I looked over the Files Changed tab
* [ ] I consulted with the ticket's product owner (if necessary)
* [x] I requested a peer review

### CODE REVIEW

File: api/src/main/java/com/staxpayments/api/models/User.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @SerialName("team_role") val teamRole: String?,
+    @SerialName("team_admin") val teamAdmin: Boolean?,
+    @SerialName("team_enabled") val teamEnabled: Boolean?,
+    @SerialName("mfa_enabled") val mfaEnabled: Boolean?,
+    @SerialName("merchant_options") val merchantOptions: List<String>?,
+    val gravatar: String?,
+    val acknowledgments: String?,
+    val options: String?,
+    val brand: String?
+)
```
Comment (muryno) [inline api/src/main/java/com/staxpayments/api/models/User.kt:28]: lint this part, you can run ktlint on the file till @thomasvidas  enforce the linting

File: api/src/main/java/com/staxpayments/api/models/User.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    @SerialName("deleted_at") val deletedAt: String?,
+    @SerialName("system_admin") val systemAdmin: Boolean?,
+    @SerialName("team_role") val teamRole: String?,
+    @SerialName("team_admin") val teamAdmin: Boolean?,
+    @SerialName("team_enabled") val teamEnabled: Boolean?,
+    @SerialName("mfa_enabled") val mfaEnabled: Boolean?,
+    @SerialName("merchant_options") val merchantOptions: List<String>?,
+    val gravatar: String?,
+    val acknowledgments: String?,
+    val options: String?,
```
Comment (thomasvidas) [inline api/src/main/java/com/staxpayments/api/models/User.kt:26]: Change the type to `JsonElement?`

**Review by thomasvidas — CHANGES_REQUESTED** [review]
A few naming changes and one type change. Overall, this is good!

File: api/src/test/java/com/staxpayments/api/models/UserTest.kt [hunk 3]
```diff
@@ -0,0 +1,68 @@
+package com.staxpayments.api.models
```
Comment (muryno) [inline api/src/test/java/com/staxpayments/api/models/UserTest.kt:1]: We done need a test case for a data class, unit test is to test a logic  @cjae

**Pull request merged**